.config/zsh/.zshrc

```diff
@@ -68,10 +68,10 @@ darwin*)
   ;;
 esac
 
-if [ -n "${commands[fzf - share]}" ]; then
-  source "$(fzf-share)/key-bindings.zsh"
-  source "$(fzf-share)/completion.zsh"
-fi
+# if [ -n "${commands[fzf - share]}" ]; then
```

Comment:
This line is the one 😠 

https://github.com/junegunn/fzf/tree/57ad21e4bd2d9c59cdfe0522183ac12c30ee2f05#fuzzy-completion-for-bash-and-zsh

Changed the integration? Looks nothing... 🤔